Haaretz.js

```diff
@@ -0,0 +1,43 @@
+{
+	"translatorID": "d6f64d96-aa6f-4fd3-816f-bdef842c7088",
+	"label": "Haaretz",
+	"creator": "Eran Rosenthal",
+	"target": "http://www\\.haaretz\\.co\\.il/",
+	"minVersion": "1.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "g",
+	"lastUpdated": "2015-08-15 13:37:56"
+}
+
+function detectWeb(doc, url) {
+	if (ZU.xpathText(doc, '//header/h1')) {
+		return 'newspaperArticle';
+	}
+}
+
+function doWeb(doc, url) {
+	var item = new Zotero.Item('newspaperArticle');
+	
+	var title = ZU.xpathText(doc, '//header/h1');
+	if(!title) {
+		title = ZU.xpathText(doc, '//meta[@property="og:title"]/@content');
+	}
+	item.title = title;
+	item.publicationTitle = 'הארץ';
+	item.url = doc.location.href;
+	item.language = 'he';
+	var abstract = ZU.xpathText(doc, '//header/p');
+	if (!abstract) abstract = ZU.xpathText(doc, '//meta[@property="og:description"]/@content');
+	item.abstractNote = abstract;
+
+	var author = ZU.xpathText(doc, '//address/a[@rel="author"]');
+	if (author) {
+		item.creators.push(author);
```

Comment:
this won't work. You need
```item.creators.push(ZU.cleanAuthor(author, "author"))``` to first generate a creator object. 

Also, does Haaretz never have multiple authors on an article? I know the English version does.